### DISCUSSION TITLE
PostgreSQL support

### DISCUSSION THREAD

**Discussion by AlexZhukovCKR** [Budibase/budibase#12363] [Help]
Good afternoon

Tell me, what version (release) of PostgreSQL does Budibase support for connecting as an external data source?

**Comment by melohagan** [comment]
Hey @AlexZhukovCKR 

This is the node library that we use to connect to Postgres: https://github.com/brianc/node-postgres

I struggled to actually find the minimum Postgres version supported anywhere in their docs.

Are you facing any connection issues yourself, or simply enquiring?

**Reply by AlexZhukovCKR** [reply]
Hello!
I am describing a technical passport where there is network interaction between Budibase and external PostgreSQL, it should indicate the versions/releases of PostgreSQL, I couldn’t find it in the Budibase documentation, so I’m wondering where to find this information.
